### PR TITLE
Fix broken readme links - Closes #189

### LINKED ISSUE #189: Some readme links are broken
### Description

- Fix broken links for Hello world and Supply chain tutorials in the readme.
- Also add new links for Postboard, Social recovery (SRS) and LNS tutorial.

### Affected Version

For SDK version `6.0.0`

### PR DESCRIPTION
**Pull request by chris529**
Fix broken links and add new links/descriptions for the Postboard, the Social recovery (SRS), and the LNS tutorial.

Closes #189

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -17,26 +17,42 @@ All industry use cases will be kept up to date with the latest Lisk SDK release.
...
 
-Originally, the Lisk Transport workshop has been designed for the **Lisk.js 2019** event.
-The project represents a flow where a carrier can sign up to pick up a packet and deliver it to the right person in a decentralized way.
+=== link:tutorials/lns[Lisk Name Service]
+A blockchain application built with the Lisk SDK v5.
+
+The LNS provides users with the ability to create a domain name service for their blockchain accounts.
+
+=== https://github.com/LiskHQ/lisk-sdk-examples/tree/v4/archive/3.x/transport[Supply chain:Lisk Transport]
+A blockchain application built with the Lisk SDK v5.
```
Comment (Tschakki) [inline README.adoc]:
```suggestion
A blockchain application built with the Lisk SDK v3.
```

File: README.adoc [hunk 2]
```diff
@@ -17,26 +17,42 @@ All industry use cases will be kept up to date with the latest Lisk SDK release.
 Also, you'll find an `/archive` folder that contains older examples referring to previous Lisk SDK releases.
 At the moment, the `/archive` folder contains examples for Lisk SDK `v2`, `v3` and `v4`.
```
Comment (Tschakki) [inline README.adoc]: The `archive` folder has been removed, so please remove this statements as well.

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: README.adoc (hunk 2 above)
Comment (chris529) [reply README.adoc]: Done 👍

File: README.adoc [hunk 3]
```diff
@@ -14,29 +14,43 @@ All code examples are grouped after the guide that contains them in the document
...
+Postboard is a Twitter-like application that offers the possibility to create/post and re-post opinions, follow other users' accounts, like posts, etc.
+
+=== link:tutorials/social-recovery[Social Recovery]
+A blockchain application built with the Lisk SDK v5.
+
+The SRS offers the functionality to enable users to build an account recovery tool to assist in retrieving lost funds.
 
-Originally, the Lisk Transport workshop has been designed for the **Lisk.js 2019** event.
-The project represents a flow where a carrier can sign up to pick up a packet and deliver it to the right person in a decentralized way.
+=== link:tutorials/lns[Lisk Name Service]
```
Comment (Tschakki) [inline README.adoc]:
```suggestion
=== link:tutorials/lisk-name-service[Lisk Name Service]
```

File: README.adoc [hunk 4]
```diff
@@ -14,29 +14,43 @@ All code examples are grouped after the guide that contains them in the document
...
 Implements the `HelloModule`, which is saving a custom string into the senders account.
-It also equipped with a basic react frontend app including the most common interactions.
+It is also equipped with a basic react frontend app including the most common interactions.
 
 === link:tutorials/nft[NFT Demo]
 A blockchain application built with the Lisk SDK v5.
 
-It demonstrates the use of On-Chain and Off-Chain architecture by creating a custom module and a plugin for Lisk application.
+It demonstrates the use of On-Chain and Off-Chain architecture by creating a custom module and a plugin for a Lisk application.
 A few transactions are also included.
```
Comment (Tschakki) [inline README.adoc]:
```suggestion
Allows a user to create, transfer and purchase NFTs.
```

File: README.adoc [hunk 5]
```diff
@@ -14,29 +14,43 @@ All code examples are grouped after the guide that contains them in the document
...
-Also, you'll find an `/archive` folder that contains older examples referring to previous Lisk SDK releases.
-At the moment, the `/archive` folder contains examples for Lisk SDK `v2`, `v3` and `v4`.
 
-=== link:tutorials/hello-world[Hello World]
+=== link:guides/01-bootstrap[Hello World]
 A hello world application built with the Lisk SDK v5.
 
 Implements the `HelloModule`, which is saving a custom string into the senders account.
-It also equipped with a basic react frontend app including the most common interactions.
+It is also equipped with a basic react frontend app including the most common interactions.
```
Comment (Tschakki) [inline README.adoc:24]:
```suggestion
It demonstrates the use of On-Chain and Off-Chain architecture by creating a custom module and a plugin for a sidechain client.

It is also equipped with a basic react frontend app including the most common interactions.
```

**Review by Tschakki — CHANGES_REQUESTED** [review]
Just a few suggestions to improve descriptions and fix link, rest LGTM :)

File: README.adoc [hunk 6]
```diff
@@ -14,29 +14,44 @@ All code examples are grouped after the guide that contains them in the document
 Each example project represents an industry use case in which we showcase the potential of the Lisk SDK and how it can transform this industry.
 
 All industry use cases will be kept up to date with the latest Lisk SDK release.
-Also, you'll find an `/archive` folder that contains older examples referring to previous Lisk SDK releases.
-At the moment, the `/archive` folder contains examples for Lisk SDK `v2`, `v3` and `v4`.
 
-=== link:tutorials/hello-world[Hello World]
+=== link:guides/01-bootstrap[Hello World]
 A hello world application built with the Lisk SDK v5.
```
Comment (TalhaMaliktz) [inline README.adoc:19]:
```suggestion
A hello world application built with the Lisk SDK v6.
```

File: README.adoc [hunk 7]
```diff
@@ -14,29 +14,44 @@ All code examples are grouped after the guide that contains them in the document
...
-It also equipped with a react frontend app to show the usage of transactions.
+Allows a user to create, transfer, and purchase NFTs.
+It is also equipped with a react frontend app to show the usage of transactions.
+
+=== link:tutorials/postboard[Postboard]
+A blockchain application built with the Lisk SDK v5.
+
+Postboard is a Twitter-like application that offers the possibility to create/post and re-post opinions, follow other users' accounts, like posts, etc.
+
+=== link:tutorials/social-recovery[Social Recovery]
```
Comment (TalhaMaliktz) [inline README.adoc]:
```suggestion
=== link:tutorials/social-recovery[Social Recovery System (SRS)]
```

File: README.adoc [hunk 8]
```diff
@@ -14,29 +14,44 @@ All code examples are grouped after the guide that contains them in the document
...
+A blockchain application built with the Lisk SDK v5.
 
-=== link:tutorials/archive/4.x/transport[Supply chain: Lisk Transport]
-A blockchain application built with the Lisk SDK v4.
+The SRS offers the functionality to enable users to build an account recovery tool to assist in retrieving lost funds.
 
-Originally, the Lisk Transport workshop has been designed for the **Lisk.js 2019** event.
-The project represents a flow where a carrier can sign up to pick up a packet and deliver it to the right person in a decentralized way.
+=== link:tutorials/lisk-name-service[Lisk Name Service]
+A blockchain application built with the Lisk SDK v5.
```
Comment (TalhaMaliktz) [inline README.adoc:43]:
```suggestion
A blockchain application built with the Lisk SDK v6.
```

File: README.adoc [hunk 9]
```diff
@@ -14,29 +14,44 @@ All code examples are grouped after the guide that contains them in the document
...
+=== link:tutorials/social-recovery[Social Recovery]
+A blockchain application built with the Lisk SDK v5.
 
-=== link:tutorials/archive/4.x/transport[Supply chain: Lisk Transport]
-A blockchain application built with the Lisk SDK v4.
+The SRS offers the functionality to enable users to build an account recovery tool to assist in retrieving lost funds.
 
-Originally, the Lisk Transport workshop has been designed for the **Lisk.js 2019** event.
-The project represents a flow where a carrier can sign up to pick up a packet and deliver it to the right person in a decentralized way.
+=== link:tutorials/lisk-name-service[Lisk Name Service]
```
Comment (TalhaMaliktz) [inline README.adoc]:
```suggestion
=== link:tutorials/lisk-name-service[Lisk Name Service (LNS)]
```

**Review by TalhaMaliktz — CHANGES_REQUESTED** [review]
Looks good :). Just some minor changes.

**Review by TalhaMaliktz — APPROVED** [review]
LGTM 😊

File: README.adoc [hunk 10]
```diff
@@ -14,29 +14,44 @@ All code examples are grouped after the guide that contains them in the document
 Each example project represents an industry use case in which we showcase the potential of the Lisk SDK and how it can transform this industry.
 
 All industry use cases will be kept up to date with the latest Lisk SDK release.
-Also, you'll find an `/archive` folder that contains older examples referring to previous Lisk SDK releases.
-At the moment, the `/archive` folder contains examples for Lisk SDK `v2`, `v3` and `v4`.
 
-=== link:tutorials/hello-world[Hello World]
-A hello world application built with the Lisk SDK v5.
+=== link:guides/01-bootstrap[Hello World]
+A hello world application built with the Lisk SDK v6.
```
Comment (Tschakki) [inline README.adoc]:
```suggestion
A hello world application built with the Lisk SDK v5.
```

File: README.adoc [hunk 11]
```diff
@@ -14,29 +14,44 @@ All code examples are grouped after the guide that contains them in the document
...
+
+Postboard is a Twitter-like application that offers the possibility to create/post and re-post opinions, follow other users' accounts, like posts, etc.
+
+=== link:tutorials/social-recovery[Social Recovery System (SRS)]
+A blockchain application built with the Lisk SDK v5.
+
+The SRS offers the functionality to enable users to build an account recovery tool to assist in retrieving lost funds.
+
+=== link:tutorials/lisk-name-service[Lisk Name Service (LNS)]
+A blockchain application built with the Lisk SDK v6.
```
Comment (Tschakki) [inline README.adoc]:
```suggestion
A blockchain application built with the Lisk SDK v5.
```

**Review by Tschakki — CHANGES_REQUESTED** [review]
The example apps are not updated yet

File: README.adoc (hunk 11 above)
Comment (chris529) [reply README.adoc]: OK,  No worries...Talha and I had a discussion about this, as we both were not sure whether to state V6 at this point. However, you are correct Mona, the original apps were created in v5, so we can always update this again, once v6 is released and the apps are updated. 👍

**Review by Tschakki — APPROVED** [review]
LGTM

**Pull request merged**